### PR TITLE
Cherry Pick PRs for (potential) final packages release for WP 6.5

### PR DESCRIPTION
**Pull request by getdave**
## What?

Contains PR cherry picked for a final (unconfirmed) packages release prior to WP 6.5.

## Why?

The bugs identified were potentially severe enough to warannt a final release of packages to ensure the fixes can land in 6.5.

## How?

Cherry Pick PRs

### CODE REVIEW

File: packages/block-editor/src/components/url-popover/image-url-input-ui.js [hunk 1]
```diff
@@ -241,14 +242,75 @@ const ImageURLInputUI = ( {
...
+			! isEditingLink
+		) {
+			return (
+				<div className="block-editor-url-popover__expand-on-click">
+					<Icon icon={ fullscreen } />
+					<div className="text">
+						<p>{ __( 'Expand on click' ) }</p>
+						<p className="description">
+							{ __( 'Scales the image with a lightbox effect' ) }
+						</p>
```
Comment (getdave) [inline packages/block-editor/src/components/url-popover/image-url-input-ui.js]: This code has been copied/moved and is _not_ a new string.

File: packages/block-editor/src/components/url-popover/image-url-input-ui.js [hunk 2]
```diff
@@ -241,14 +242,75 @@ const ImageURLInputUI = ( {
...
+					</div>
+					<Button
+						icon={ linkOff }
+						label={ __( 'Disable expand on click' ) }
```
Comment (getdave) [inline packages/block-editor/src/components/url-popover/image-url-input-ui.js]: This code has been copied/moved and is not a new string.

**Comment by getdave** [comment]
~Still waiting on https://github.com/WordPress/gutenberg/pull/60130~

✅ Picked

File: packages/block-library/src/navigation/view.js [hunk 3]
```diff
@@ -66,7 +66,14 @@ const { state, actions } = store(
 					actions.openMenu( 'hover' );
 			},
 			closeMenuOnHover() {
-				actions.closeMenu( 'hover' );
+				const { type, overlayOpenedBy } = getContext();
+				if (
+					type === 'submenu' &&
+					// Only open on hover if the overlay is closed.
```
Comment (swissspidy) [inline packages/block-library/src/navigation/view.js]: This comment was missed:

https://github.com/WordPress/gutenberg/pull/60130#discussion_r1537208503

Also a lack of braces here, which is usually error prone and might be against the coding standards too

Comment (getdave) [reply packages/block-library/src/navigation/view.js]: Good spot. It's frustrating that that code formatting issue wasn't picked up by linting. Going to raise a PR to fix this one.

Comment (getdave) [reply packages/block-library/src/navigation/view.js]: https://github.com/WordPress/gutenberg/pull/60162

Comment (draganescu) [reply packages/block-library/src/navigation/view.js]: I was annoyed at the lack of braces too but I wanted to keep the same snippet that we have above for the other handler.

**Comment by getdave** [comment]
Going to work on a new branch

**Pull request closed without merging**